bench-vortex/src/bin/notimplemented.rs

```diff
@@ -188,8 +188,6 @@ fn compute_funcs(encodings: &[ArrayData]) {
             "search_sorted",
             "slice",
             "take",
-            "and",
```

Comment:
Sadly we can't probe for support when we need to check the input arrays first...

There might be a better model for this, e.g. `binary_boolean_fn() -> Option<BooleanFn>` where `BooleanFn { fn boolean(...) -> Result<Option<ArrayData>>`

But in this model, we have to _try_ to execute the function to see if it exists, meaning we can't probe two different encodings without the first one performing the computation. I'm not sure we ever have that case right now, but it's something to think about.

Reply:
We do the thing for compare function but that produces 3d tables which are hard to display

Reply:
I have a FLUP issue. I think if an encoding implements _any_ CompareFn, then we should just say it's covered for now.